### PR TITLE
Update cisco-all-devices.yml

### PR DESCRIPTION
**Pull request by aditdeshpande10**
Adding OIDs for Session Timeout, Reauth enabled, Reauth interval

### CODE REVIEW

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml [hunk 1]
```diff
@@ -189,6 +190,7 @@ metrics:
         column:
           name: cefcPowerUnits
           OID: 1.3.6.1.4.1.9.9.117.1.1.1.1.2
+      #
```
Comment (thezackm) [inline profiles/kentik_snmp/cisco/cisco-all-devices.yml:193]: why is this comment line in here?

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml [hunk 2]
```diff
@@ -1146,5 +1148,25 @@ metrics:
...
+    table:
+      OID: 1.3.6.1.4.1.9.9.656.1.2.1
+      name: cafPortConfigTable
+    symbols: 
+        # Specifies if reauthentication is enabled for this port.
+      - OID: 1.3.6.1.4.1.9.9.656.1.2.1.1.6
+        name: cafPortReauthEnabled
+        # Specifies the reauthentication interval, after which the port will be reauthenticated if value of the corresponding instanceof cafPortReauthEnabled is 'true'
+      - OID: 1.3.6.1.4.1.9.9.656.1.2.1.1.7
+        name: cafPortReauthInterval
```
Comment (thezackm) [inline profiles/kentik_snmp/cisco/cisco-all-devices.yml:1170]: this isn't formatted correctly for a tabular MIB; there should be a single table with symbols that will be the numeric metric values; and optionally metric_tags that will be decorated attributes on the aforementioned metrics.

**Review by thezackm — CHANGES_REQUESTED** [review]
@aditdeshpande10 - thanks for the submission, please see comments

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml (hunk 1 above)
Comment (aditdeshpande10) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml]: I've removed it, seems like a typo.

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml (hunk 2 above)
Comment (aditdeshpande10) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml]: I've used metric_tags for the OID which isn't in the table.

**Review by thezackm — CHANGES_REQUESTED** [review]
@aditdeshpande10 - one more small edit to make sure we get the proper "true|false" string from the device.

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml [hunk 3]
```diff
@@ -1146,5 +1147,23 @@ metrics:
...
           conversion: hextoip
+    - MIB: CISCO-AUTH-FRAMEWORK-MIB 
+    # A list of port entries. An entry will exist for each interface which support Authentication Framework feature."       
+    table:
+      name: cafPortConfigTable
+      OID: 1.3.6.1.4.1.9.9.656.1.2.1
+    symbols: 
+        # Specifies if reauthentication is enabled for this port.
+      - name: cafPortReauthEnabled      
+        OID: 1.3.6.1.4.1.9.9.656.1.2.1.1.6
```
[the remark this answers is not in the document]
Comment (aditdeshpande10) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml:1158]: @thezackm I've added the enum for the aforementioned metric.

**Review by thezackm — APPROVED** [review]

**Pull request merged**